README.adoc

```diff
@@ -1,15 +1,62 @@
 = keep-core
 
-The core code behind the https://keep.network[Keep network].
+https://circleci.com/gh/keep-network/keep-core[image:https://circleci.com/gh/keep-network/keep-core.svg?style=svg&circle-token=ec728f5ca814b6cb2db5ffeb7258151b752a207e[CircleCI Build Status]]
+https://discord.gg/wYezN7v[image:https://img.shields.io/badge/chat-Discord-blueViolet.svg[Chat with us on Discord]]
 
-== Getting Started as a Developer
+The core contracts and reference client implementation behind the Keep network,
+a privacy, interoperability, and censorship-resistance toolkit for developers
+on Ethereum.
 
-Start with the README in link:docs/development/[the developer docs directory].
+== What's a keep?
 
-== Directory structure
+The network offers application developers **keeps**, small off-chain data
+containers for private storage and computation that can be opened, closed, and
+managed by smart contracts autonomously.
 
-The directory structure used in the Keep repository is very similar to that used
-in other Go projects:
+Keeps are maintained by stakers, actors who run nodes and have skin in the
+game. When a new keep is opened, the requisite number of stakers are chosen
+via a BLS-based random beacon to maintain the keep.
+
+The first type of keep launching with the network is the `BondedECDSAKeep`,
```

Comment:
"first type of Keep" I like the phrasing here, sets up the mental model for the AESKeep, etc.

Link out to keep-ecdsa here? Or perhaps below in dApp Developers.

Reply:
Porque no los dos?